pkg/scanners/trivy/trivy.go

```diff
@@ -241,8 +242,10 @@ func initScanner(ctx context.Context) (Scanner, error) {
 func scan(s Scanner, allImages []eraserv1alpha1.Image) ([]eraserv1alpha1.Image, []eraserv1alpha1.Image) {
 	vulnerableImages := make([]eraserv1alpha1.Image, 0, len(allImages))
 	failedImages := make([]eraserv1alpha1.Image, 0, len(allImages))
+	// track total scan job time
```

Comment:
You can create a `time.Timer` and use a `select` here. It will be quite a bit easier than manually measuring each time through the loop.